extensions/amp-form/0.1/amp-form-textarea.js

```diff
@@ -77,7 +77,7 @@ export class AmpFormTextarea {
     this.doc_ = (root.ownerDocument || root);
 
     /** @private @const */
-    this.win_ = devAssert(this.doc_.defaultView);
+    this.win_ = /** @type {!Window} */ (devAssert(this.doc_.defaultView));
```

Comment:
can this be on the declaration instead?

Reply:
The problem is that `devAssert` can return `null`. Without this change:
```js
extensions/amp-form/0.1/amp-form-textarea.js:80: ERROR - assignment to property win_ of AmpFormTextarea
found   : (Window|null)
required: Window
    this.win_ = devAssert(this.doc_.defaultView);
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```